framework/web/js/source/jquery.ba-bbq.min.js

```diff
@@ -1,18 +1,17 @@
 /*
```

Comment:
Perhaps add an addition comment that says:

```
Manually re-uglified with `uglifyjs jquery.ba-bbq.js --compress -o jquery.ba-bbq.min.js`
```

Reply:
I don't think that such information should be in minified version. If we want to document how to minify it it the future, it should be rather in source file.